### DISCUSSION TITLE
Table: Allow the sortorder to be set on column basis and turn off bi-directional sorting as a parameter

### DISCUSSION THREAD

**Discussion by nicolaspierre1990** [primefaces/community#2227] [PrimeNG]
A table and his columns that can be sorted could be set on the `pSortableColumn` or `p-sortIcon` so we can force users to for example only sort in a descending way and not both ways

**Comment by rcourtman** [comment]
Fixed in commit a6bd05ce.

The SMART disk temperature data was being collected by the agent but never passed through to the frontend due to a bug in the data conversion pipeline. The fix includes:

1. Backend now properly passes SMART data to the frontend
2. The temperature cell now shows disk temperatures in its tooltip under a "Disk Temperatures" section
3. If no CPU/sensor temperatures are available, disk temperatures will be shown as the primary temperature

This will be included in the next release.
